### ISSUE TITLE
[FEATURE REQUEST] [IDEA] Add Localization support. Using browsers i18n.

### ISSUE THREAD

**Issue by johnpaulmedina** [laravel/jetstream#610]
Here is a quick example using the Jetstream inertia stack. Keep's the same style that blade uses currently including support for attribute params. I am currently using it with automatic detection of users locale. I'm sure you guys can create something much better to add out of the box support keeping that same feel you already have on Laravel with the blade templates.

So if you had `lang/en/team.php`
```php
...
'Create Team' => 'Create Organization',
...
```
```js
{{ __('team.Create Team') }} // Create Organization
```

For Vue.js driven apps - Add to app.js
```js
Vue.prototype.__ = (string, attributes) => {
    let value = _.get(window.i18n, string);
    _.eachRight(attributes, (v, k) => {
        value = _.replace(value, `:${k}`, v);
    });
    return value ? value : string;
};
```

Add to Laravel web.php routes
```php
Route::get('/assets/lang/{locale}.js', function($locale = 'en') {
    $translations = Cache::rememberForever("lang:{$locale}", function () use ($locale) {
            $files = glob(resource_path("lang/{$locale}/*.php"));
            $translations = [];

            foreach ($files as $file) {
                $name = basename($file, '.php');
                $translations[$name] = require $file;
            }
            
            return json_encode($translations);
     });
    
     return response("window.i18n = {$translations};")->header('Content-Type', 'text/javascript');
})->name('assets.lang');
```

Add this to your main app.blade.php Template
```html
<script src="{{ route('assets.lang', ['locale' => 'en']) }}"></script>
```

**Comment by driesvints** [comment]
We actually removed localisation from Inertia recently again and won't be reconsidering, sorry. Please also try to search for previous issues/prs. Thanks.

**Comment by johnpaulmedina** [comment]
Just for my own understanding if you wouldn't mind sharing - why remove it? Also, I did search for it in the issues but not prs. 

In regards to what I had found it was a separate package and used the localization differently than that of regular Laravel blade syntax. 

The code above gives the exact same feel across the board for vue.js inertia and Livewire uses the blade syntax already so really there is no difference. 

I made my own package internally and it is a little different then that above to get more flexibility so I am not in "need" of it but it seem's like something great taken out of the main app and leaving most end-users to rely on creating their own and/or using an additional package.
